docs/source/introduction.rst

```diff
@@ -1,59 +1,20 @@
 Introduction
-------------
+============
 
-The Formulations and Computational Engineering (FORCE) project, implemented within the EU Horizon
-2020 program (https://www.the-force-project.eu), aims to bring materials modelling to the heart of
-business decision making via creation of Business Decision Support Systems (BDSS).
-By doing so, industries will be able to make more knowledgeable decisions based not only on existing
-legacy data, but also on new data generated by state of the art multi-scale and multi-physics materials
-modelling.
+The business decision support system (BDSS) is a software for optimization of industrial processes
+in terms of economic and material constraints. It has been developed under the aegis of the
+`Formulations and Computational Engineering <https://www.the-force-project.eu/>`_ (FORCE) project
+of the `European Materials Modelling Council <https://emmc.info/>`_ (EMMC). The EMMC is a consortium
+of academic and industrial partners spread across Europe,
+with the mission of "the integration of materials modelling and digitalization critical for more
+agile and sustainable product development."
 
-This software provides an integrated framework that connects simulation
-models of varying complexity, experimental data sets and commercially relevant information
-(material / production costs) together as a ``Workflow`` pipeline construct. Consequently, users are
-able to select from a wide variety of parameters and key performance indicators (KPIs) that can be used
-to identify the optimal trade off when making a business decisions. An integrated set of multi-criteria
-optimization (MCO) algorithms will then identify optimal outcomes to consider, typically by mapping out a
-Pareto frontier.
+More specifically BDSS is a software for single- and multiple-criterion optimization of a directed
```

Comment:
Should we be using something else then camel case for "civilians"?

Reply:
Probably, but I guess at the moment since there's not an agreed abbreviation for the "Force Workflow Manager", it might be less confusing just to use what already exists in the source code documentation.